Maquetado/scss/details.scss

```diff
@@ -4,10 +4,26 @@
   flex-wrap: wrap;
 }
 
+@mixin shadow {
+  box-shadow: 4px 4px 10px $black-transparent;
+}
+
+@mixin text-format($size,$weight,$height) {
+  font-size: $size;
+  font-weight: $weight;
+  line-height: $height;
```

Comment:
Remember the order of the properties

Reply:
that's because it starts with Font and Line, should i change it?